pycloudlib/gce/cloud.py

```diff
@@ -104,13 +106,14 @@ def __init__(
                     raise CloudSetupError(exception_text)
                 project = result.stdout
 
-        # disable cache_discovery due to:
-        # https://github.com/google/google-api-python-client/issues/299
-        self.compute = googleapiclient.discovery.build(
-            "compute",
-            "v1",
-            cache_discovery=False,
-            credentials=credentials,
+        self.images = compute_v1.ImagesClient(credentials=credentials)
+        self.disks = compute_v1.DisksClient(credentials=credentials)
+        self.instances = compute_v1.InstancesClient(credentials=credentials)
+        self.zone_operations = compute_v1.ZoneOperationsClient(
+            credentials=credentials
+        )
+        self.global_operations = compute_v1.GlobalOperationsClient(
```

Comment:
for clarity, could we rename these to all end with "_client" ?

Reply:
also nit/question: would it make sense to rename these to start with a "_" to denote them as "private"?
For the two ibm clouds, i kept the "client" variables as private but many other clouds don't do this so this is by no means a well defined standard or rule.

Reply:
I can rename to append "_client" for all the applicable class attributes.

Can't see an issue with making them private as well, so this refactor is a good time to do that as well.